python/cugraph_service/README.md

```diff
@@ -13,18 +13,25 @@
 
     TBD
 
-### Example
-Starting the server
-```
-$> PYTHONPATH=./python/cugraph_service python -m cugraph_service_server.server
-```
-
 ## Client
 (description)
 ### Installing the `cugraph_service_client` conda package
 
     TBD
 
+### Example
+Starting a server for single-GPU-only cuGraph, using server extensions in `/my/cugraph_service/extensions`:
+```
+$> PYTHONPATH=/Projects/cugraph/python/cugraph_service python -m cugraph_service_server.server --graph-creation-extension-dir=/my/cugraph_service/extensions
```

Comment:
I suggest using an environment variable to set path and not have hard_coded path here.